src/tgen-graph.c

```diff
@@ -392,7 +392,7 @@ static GError* _tgengraph_parseGraphEdges(TGenGraph* g) {
     if(!error) {
         g->edgeCount = igraph_ecount(g->graph);
         if(g->edgeCount != edgeCount) {
-            tgen_warning("igraph_vcount %f does not match iterator count %f", g->edgeCount, edgeCount);
+            tgen_warning("igraph_vcount %d does not match iterator count %d", g->edgeCount, edgeCount);
```

Comment:
In old versions of igraph, `igraph_integer_t` is actually a `double`, so this might be a warning on older distros. But maybe all of our shadow-supported platforms now use more recent versions of igraph, so that would be a non-issue for us.